HousingManagementSystemApi.Tests/UseCasesTests/RetrieveAddressesUseCaseTests.cs

```diff
@@ -44,13 +56,16 @@ public async Task GivenAPostcode_WhenExecute_GatewayReceivesCorrectInput()
         public async Task GivenAPostcode_WhenAnAddressExistsWithAnEligibleAssetType_ThenThePropertyIsReturned()
         {
             const string TestPostcode = "postcode";
-            retrieveAddressesGateway.Setup(x => x.SearchByPostcode(TestPostcode))
+            this.retrieveAddressesGateway.Setup(x => x.SearchByPostcode(TestPostcode))
                 .ReturnsAsync(new PropertyAddress[] { new() { PostalCode = TestPostcode, Reference = new Reference { ID = "assetId" } } });
 
-            retrieveAssetGateway.Setup(x => x.RetrieveAsset("assetId"))
-                .ReturnsAsync(new AssetResponseObject { AssetType = AssetType.Dwelling });
+            this.retrieveAssetGateway.Setup(x => x.RetrieveAsset("assetId"))
+                .ReturnsAsync(new AssetResponseObject { AssetType = AssetType.Dwelling, Tenure = new AssetTenureResponseObject { Id = new Guid().ToString() } });
 
-            var result = await retrieveAddressesUseCase.Execute(TestPostcode);
+            this.tenureGateway.Setup(x => x.RetrieveTenureType(new Guid().ToString()))
```

Comment:
Instead of creating new GUIDs and converting to a string, create strings instead:
```suggestion
            this.tenureGateway.Setup(x => x.RetrieveTenureType("Id"))
```

Multiple occurrences of this.